content/en/docs/concepts/overview/working-with-objects/names.md

```diff
@@ -42,6 +69,7 @@ spec:
     - containerPort: 80
 ```
 
+
```

Comment:
Another nit:
Is the Pod manifest a good example of naming conventions or could you provide some different examples of
Kubernetes resource names.

Reply:
Pod is the basic building block in Kubernetes, so I think keeping it here makes sense.
We can certainly provide more examples here or somewhere else but I don't see it necessary.
In follow-up PRs, there are references to these resource name constraints.